### ISSUE TITLE
Latest security update breaks theme editing, likely over-escaping

### ISSUE THREAD

**Issue by luap42** [contao/contao#3309]
**Affected version(s)**

Contao 4.11.8 latest update (with the security update published on 2021-08-11, https://contao.org/de/news/sicherheitsupdates-am-11-august-2021.html)

**Description**

Before: You have a theme viewport setting of `width=device-width, initial-scale=1.0` (or anything similar with a `=` sign). The site works as intended.

Then: You make any change to the additional `<head>` content field (or possibly any other field, but not tested for that behavior).

After: The theme viewport setting looks equal, but the site broke. You can see in the revision history that `=` was replaced with its HTML entity ``&#61;`:

![revision history excerpt](https://user-images.githubusercontent.com/21335202/129483275-a1913255-19db-4bdd-acfc-4cf36e1b6f13.png)

This bug has been introduced by one of the recent security updates, because editing worked fine before and in pre-updated contao installations. Without deeper knowledge to contao's working, I'd guess that the security update introduced a feature escaping more input, which is a little bit too eager escaping values it shouldn't.

**Comment by fritzmg** [comment]
Basically duplicate of #3306

**Comment by ausi** [comment]
@luap42 can you verify if the changes from #3315 fix your issue?
